webapp/sources/rudder/rudder-web/src/main/elm/rules/sources/ApiCalls.elm

```diff
@@ -104,6 +104,22 @@ getRuleDetails model ruleId =
   in
     send GetRuleDetailsResult req
 
+getRulesCategoryDetails : Model -> String -> Cmd Msg
```

Comment:
Why do you changes on RUles Category here ?

Reply:
I forgot to specify this change. For several reasons:
- To have the same behavior as when selecting a rule
- To be sure to be up to date when opening the details of a category
- And above all, to be able to retrieve the information of a category only from an ID (since this is the only information we get when we parse the URL)